### PR TITLE
Turn off unnecessary processing and add doc links.

### PR DESCRIPTION
**Pull request by qingling128**

### CODE REVIEW

File: agents.yaml [hunk 1]
```diff
@@ -437,10 +437,11 @@ metadata:
   labels:
     app: stackdriver-log-forwarder
 data:
-  # Configuration files: server, input, filters and output
+  # Configuration files for service, input, filters and output plugins.
```
Comment (jkschulz) [inline agents.yaml]: `filters and` → `filter, and`

File: log-forwarder.yaml [hunk 2]
```diff
@@ -106,21 +108,25 @@ data:
 
   filter-kubernetes.conf: |
     [FILTER]
+        # https://docs.fluentbit.io/manual/filter/kubernetes
         Name                kubernetes
         Match               k8s_container.*
         Kube_URL            https://kubernetes.default.svc.cluster.local:443
         Merge_Log           Off
-        K8S-Logging.Parser  On
+        K8S-Logging.Parser  Off
```
Comment (jkschulz) [inline log-forwarder.yaml]: This and `Merge_Log` are already Off by default so you can remove those lines (unless you want it explicit)

File: agents.yaml (hunk 1 above)
Comment (qingling128) [reply agents.yaml]: Oops. Changed.

File: log-forwarder.yaml (hunk 2 above)
Comment (qingling128) [reply log-forwarder.yaml]: Ah, right. Removed.

**Pull request merged**